### DISCUSSION TITLE
Login UI "Bad credentials, 400: Bad request" after restore

### DISCUSSION THREAD

**Discussion by Zach-Harald** [amidaware/tacticalrmm#1401] [General]
Hallo,

After a restore to a new server, I can no longer log in to the web interface.
I get the message "Bad credentials, 400: Bad request".

I have performed the restore as described in the documentation.
On the old server I updated Tactical RMM with the latest version, then I made a backup.
On the new server I restored the backup with the restore routine.
After the restore I got a message that the restore was successful.
A login on the web UI on the new server brought this error message.

I changed all three A-records (rmm, api, mesh) to the new IP addresses.

Who can help me?

**Comment by dinger1986** [comment]
Did you test the old server before backing up? What version were you on?

Do you use discord?

can you run through https://docs.tacticalrmm.com/troubleshooting/

**Comment by Zach-Harald** [comment]
I tested the old server, everything was ok.

Version of Tactical RMM on old server is V 0.15.6

 

What is discord?

 

I run the troubleshootingscript, see the log in attachement.

 

 

 

 

Mit freundlichen Grüßen

 

Dipl.-Ing. (FH)

Harald Zach

Nadistr. 20

80809 München

 

Telefon : (089) 99 02 03 40

Fax        : (089) 99 02 03 42

Mobil     : (0171) 9 38 19 36

Internet :  <https://it-tataa.de/> https://zach1.de

 

Von: dinger1986 ***@***.***> 
Gesendet: Donnerstag, 12. Januar 2023 12:10
An: amidaware/tacticalrmm ***@***.***>
Cc: Zach-Harald ***@***.***>; Author ***@***.***>
Betreff: Re: [amidaware/tacticalrmm] Login UI "Bad credentials, 400: Bad request" after restore (Discussion #1401)

 

Did you test the old server before backing up? What version were you on?

Do you use discord?

can you run through https://docs.tacticalrmm.com/troubleshooting/

**Comment by dinger1986** [comment]
theres no attachment, please add it directly not email a reply

**Reply by Zach-Harald** [reply]
[checklog.log](https://github.com/amidaware/tacticalrmm/files/10401501/checklog.log)

**Comment by dinger1986** [comment]
whats the output of `cat /etc/hosts`

**Reply by Zach-Harald** [reply]
<img width="1121" alt="hosts" src="https://user-images.githubusercontent.com/81093130/212067953-1b8da876-4562-4846-9924-a69821ae18cb.png">

**Comment by dinger1986** [comment]
at what point do you get the error? does it prompt for mfa code?

**Reply by Zach-Harald** [reply]
The normal login page appear. The error appears after entering user and password, no mfa box appears.

**Reply by dinger1986** [reply]
just in case can you check this? https://docs.tacticalrmm.com/troubleshooting/#cant-login-on-server-after-first-setup

**Reply by Zach-Harald** [reply]
i shoudt check the time? clienet and server time are the same.

**Reply by Zach-Harald** [reply]
Still getting this:

<img width="953" alt="Bad Request" src="https://user-images.githubusercontent.com/81093130/212312905-11f4e31c-b25c-40d1-9f08-2ac4c6ecacd4.png">

**Reply by dinger1986** [reply]
its maybe worth doing a new backup and restore, maybe this one has gone sideways

**Reply by silversword411** [reply]
Where are you hosting the server?

**Reply by Zach-Harald** [reply]
I made a complete new restore. Now it works perfectly. Thanks for helping.

**Reply by dinger1986** [reply]
> I made a complete new restore. Now it works perfectly. Thanks for helping.

Great glad too hear that!
